<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>einstellungen_seite</class>
 <widget class="QWidget" name="einstellungen_seite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>System-ui</family>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Heu-Fütterung</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fdffe0;</string>
  </property>
  <widget class="QLabel" name="label_gewicht">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>label_gewicht</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_simulation_toggle">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>490</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Simulation an/aus</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_kalibrieren">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kalibrierung der Waage</string>
   </property>
  </widget>
  <widget class="QLabel" name="ub_einstellungen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F;
color: #fdffe0;
</string>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_kalibrieren">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2E672F;
    color: #ffffff;
    border-radius: 15px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/Calib.png</normaloff>
     <normalon>:/icons/Calib.png</normalon>:/icons/Calib.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2E672F;
    color: #ffffff;
    border-radius: 15px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/Display.png</normaloff>
     <normalon>:/icons/Display.png</normalon>:/icons/Display.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_simulation_toggle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: lightgray;
    color: black;
    border-radius: 15px;
    border: none;
}
QPushButton:checked {
    background-color: red;
    color: white;
    border-radius: 15px;
    border: none;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
    border-radius: 15px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/Sim.png</normaloff>:/icons/Sim.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/Back.png</normaloff>:/icons/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_we_auswahl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2E672F;
    color: #ffffff;
    border-radius: 15px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/Calib.png</normaloff>
     <normalon>:/icons/Calib.png</normalon>:/icons/Calib.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txt_we_auswahl">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Wiegeeinheit auswählen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_aktuelle_we">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">label_aktuelle_we</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sensor_status">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">label_sensor_status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fu_sim_toggle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: lightgray;
    color: black;
    border-radius: 15px;
    border: none;
}
QPushButton:checked {
    background-color: red;
    color: white;
    border-radius: 15px;
    border: none;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
    border-radius: 15px;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/Sim.png</normaloff>:/icons/Sim.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>ub_einstellungen</zorder>
  <zorder>label_gewicht</zorder>
  <zorder>txt_simulation_toggle</zorder>
  <zorder>txt_kalibrieren</zorder>
  <zorder>btn_kalibrieren</zorder>
  <zorder>pushButton</zorder>
  <zorder>btn_simulation_toggle</zorder>
  <zorder>btn_back</zorder>
  <zorder>btn_we_auswahl</zorder>
  <zorder>txt_we_auswahl</zorder>
  <zorder>label_aktuelle_we</zorder>
  <zorder>label_sensor_status</zorder>
  <zorder>btn_fu_sim_toggle</zorder>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
